cpp/ps0003/ps0003_sol.cpp

```diff
@@ -4,14 +4,17 @@ using namespace std;
 
 int main()
 {
-    short s ;
-    cin >> s ;
-     if ( s % 2 == 0 ) {
+    // prompting user for number 2/2
+    short s;
+    cin >> s;
+
+    // checking if the number is odd or even 3/3
+    if (s % 2 == 0) {
         cout << "Number is even \n" ;
-     } else {
+    } else {
         cout<< "Number is odd \n" ;
-     }
+    }

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console